src/plugins/opensearch_dashboards_react/public/overview_page/overview_page_footer/overview_page_footer.tsx

```diff
@@ -67,7 +67,12 @@ export const OverviewPageFooter: FC<Props> = ({ addBasePath, path }) => {
         >
           <FormattedMessage
             id="opensearch-dashboards-react.pageFooter.changeHomeRouteLink"
-            defaultMessage="Display a different page on log in"
+            defaultMessage={i18n.translate(
+              'opensearch-dashboards-react.pageFooter.changeHomeRouteLink',
```

Comment:
Is there an existing paradigm for testing the translations that we can apply to these changes?

Reply:
Could not find it when I first search. Thinking might be from some older version? Then think this also needs to clean to make the whole page look nicer. Will take more time to see whether I could find it.